### PR TITLE
Reload redis scripts on reconnect

### PR DESCRIPTION
**Pull request by bp-FLN**
this is basically the same as #143, but for `WorkerPoolImpl`

### CODE REVIEW

File: src/main/java/net/greghaines/jesque/worker/DefaultPoolExceptionHandler.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+	public RecoveryStrategy onException(final JobExecutor jobExecutor, final Exception exception, 
+	        final String curQueue) {
+		if (exception instanceof JedisConnectionException
+				|| exception instanceof JedisNoScriptException
+				|| (exception instanceof JedisDataException && exception.getMessage().equals("LOADING Redis is loading the dataset in memory"))) {
```
Comment (gresrun) [inline src/main/java/net/greghaines/jesque/worker/DefaultPoolExceptionHandler.java]: Can you break this long line?

File: src/main/java/net/greghaines/jesque/worker/WorkerPoolImpl.java [hunk 2]
```diff
@@ -557,7 +556,16 @@ protected void recoverFromException(final String curQueue, final Exception ex) {
         final RecoveryStrategy recoveryStrategy = this.exceptionHandlerRef.get().onException(this, ex, curQueue);
         switch (recoveryStrategy) {
         case RECONNECT:
-            LOG.info("Ignoring RECONNECT strategy in response to exception because this is a pool", ex);
+            if (ex instanceof JedisNoScriptException) {
+                LOG.info("got JedisNoScriptException while reconnecting, reloading redis scripts");
```
Comment (gresrun) [inline src/main/java/net/greghaines/jesque/worker/WorkerPoolImpl.java]: Nit: Capital 'G' for start of sentence. Also, Redis should be capitalized.

File: src/main/java/net/greghaines/jesque/worker/WorkerPoolImpl.java [hunk 3]
```diff
@@ -557,7 +556,16 @@ protected void recoverFromException(final String curQueue, final Exception ex) {
...
+                loadRedisScripts();
+            } else {
+                LOG.info("waiting for pool to reconnect to redis", ex);
```
Comment (gresrun) [inline src/main/java/net/greghaines/jesque/worker/WorkerPoolImpl.java]: Same. Also, consider adding the duration of the wait to the message.

File: src/main/java/net/greghaines/jesque/worker/DefaultPoolExceptionHandler.java (hunk 1 above)
Comment (bp-FLN) [reply src/main/java/net/greghaines/jesque/worker/DefaultPoolExceptionHandler.java]: done (i actually moved it to a separate variable, as breaking inside the if was making it even more unreadable)

File: src/main/java/net/greghaines/jesque/worker/WorkerPoolImpl.java (hunk 2 above)
Comment (bp-FLN) [reply src/main/java/net/greghaines/jesque/worker/WorkerPoolImpl.java]: done

File: src/main/java/net/greghaines/jesque/worker/WorkerPoolImpl.java (hunk 3 above)
Comment (bp-FLN) [reply src/main/java/net/greghaines/jesque/worker/WorkerPoolImpl.java]: done

**Comment by sar009** [comment]
@gresrun any idea when this fix will be available in maven.

**Pull request merged**